src/menu.ts

```diff
@@ -27,6 +28,9 @@ const menuTemplate: MenuItemConstructorOptions[] = [
             { role: 'reload' },
             { role: 'forceReload' },
             { role: 'toggleDevTools' },
+            { type:"checkbox","label":"Toggle Menu Bar", click(event:any){
```

Comment:
I don't think you need `any` here. If you remove that entirely, TypeScript should be able to infer the event type correctly.

Nitpicking, but: it's also be nice to add some spaces in here, to make the codestyle match the rest of this file.